transport-native-unix-common/src/main/c/netty_unix_socket.c

```diff
@@ -671,6 +683,15 @@ static jbyteArray netty_unix_socket_remoteAddress(JNIEnv* env, jclass clazz, jin
     return netty_unix_socket_createInetSocketAddressArray(env, &addr);
 }
 
+static jbyteArray netty_unix_socket_remoteDomainSocketAddress(JNIEnv* env, jclass clazz, jint fd) {
+    struct sockaddr_storage addr;
```

Comment:
Something I just noticed (we can fix this in another PR; affects other functions as well) is that we should probably initialize `addr` to zero, since we don't pass along the `len` output, so `strlen` will end up reading uninitialized memory.